en/server-side/nodejs.md

```diff
@@ -26,6 +26,20 @@ This should print the version number of Node.js like this:
 ```bash
 v20.7.0
 ```
+## Running the Node.js REPL
+
+REPL stands for Read Evaluate Print Loop, and allows you to run code from the terminal.
+Type into your terminal this command:
+
+```bash
+node
+```
+In this prompt, you can run code, like this:
+
+```js
+> console.log("Hello World")
+```
+You can leave the REPL by typing CTRL + d
```

Comment:
I believe it should be `ctrl-C` two times as described [here](https://nodejs.org/en/learn/command-line/how-to-use-the-nodejs-repl).

Reply:
Ok. Was not aware of that.